.github/workflows/pr_basic_compilation_check.yml

```diff
@@ -5,7 +5,7 @@ on: [push, pull_request]
 jobs:
   basic_complication_check:
     # Run all steps in the compilation testing containers
-    container: runetest/compilation-testing-librats:anolis8.4
+    container: runetest/compilation-testing:anolis8.4
```

Comment:
The previous docker image `runetest/compilation-testing-librats` was missing some components (`patch` command [[log1]](https://github.com/KB5201314/librats/actions/runs/3358848091/jobs/5566287943#step:6:84), and mitigration tools required by sgx-ssl [[log2]](https://github.com/KB5201314/librats/actions/runs/3358795886/jobs/5566184961#step:7:303)).

This commit is just a temporary workaround, and I will revert this commit after the docker image is updated.

Reply:
Nice work. You can submit another pr about ci/cd fix. 

For example,
1. you can update the `runetest/compilation-testing-librats` Dockfile to install mitigration tools etc.
2. Then use the [workflow](https://github.com/inclavare-containers/librats/actions/workflows/manually_compilation_testing_image.yml) to push
 the newest docker image to docker hub
3. We can direct merge the ci/cd new pr.

Reply:
resolved https://github.com/inclavare-containers/librats/actions/runs/3573801545/workflow